dotcom-rendering/src/web/experiments/tests/recipe-structured-data.ts

```diff
@@ -0,0 +1,68 @@
+// This is for an experiment testing SEO performance on recipes before
+// and after structured data is added. Please don't delete - it will be
+// removed once the the experiment is finished, we swear!
+
+export type RecipeSchemaObject = {
```

Comment:
```suggestion type RecipeSchemaObject = {
```

I can't see this type being used elsewhere so I think we're safe not exporting it